graylog2-web-interface/src/views/components/views/ShareViewModal.jsx

```diff
@@ -135,7 +138,7 @@ class ShareViewModal extends React.Component<Props, State> {
           Any user of this Graylog
         </Radio>{' '}
         <Additional>
-          <HelpBlock>Anyone with an account can access the dashboard.</HelpBlock>
+          <HelpBlock>Anyone with an account can access the {view.typeLabel}.</HelpBlock>
```

Comment:
We could also replace this with a `<ViewTypeLabel type={view.type} />` that renders the view type appropriately. This would also remove the need for a getter in the data class (and solve a presentation issue in the presentation layer).

Reply:
I like the idea and implemented the approach. What bothers me a little, the component is just a function which returns a string. The main reason I am using the `<ViewTypeLabel type={view.type} />` syntax is the import from `views/components`. While the syntax is useful in most cases a string would be easier to handle in this case, e.g. for the modal title.